vault/dendron.topic.images.md

```diff
@@ -2,35 +2,42 @@
 id: a91fd8da-6895-49fe-8164-a17acd8d9a17
 title: Images
 desc: ''
-updated: 1659026813451
+updated: 1662866892933
 created: 1595562035825
-stub: false
 ---
-    
-## Copy & Paste
+## Summary
 
-Dendron automatically inserts images in your clipboard using the `> Paste Image` command. The image is saved to `${currentFileDir}/assets` by default. This feature requires the [Paste Image](https://link.dendron.so/vscode-paste-image) VS Code extension.
+Dendron extends regular Markdown image support by allowing custom CSS properties. Images can also be pasted into a file from your clipboard.
+
+## Use Cases
+
+* add images and screenshots to notes
+* modify images using inline YAML map
+
+## Features
+### Copy & Paste
+
+Dendron automatically inserts images in your clipboard using the `> Paste Image` command. The image is saved to `${currentFileDir}/assets/images` by default. This feature requires the [Paste Image](https://link.dendron.so/vscode-paste-image) VS Code extension.
```

Comment:
the copy here is a little outdated. features shouldn't describe "how to use" the functionality but rather answer "jobs to be done". see https://handbook.dendron.so/notes/q2c5zysqy12zyij528m7uqc#features

would change the content here to a `Getting Started` section at the bottom instead